WhatsInSeason/README.Rmd

```diff
@@ -0,0 +1,26 @@
+---
+title: "What's in Season?"
+author: "Annie Johansson"
+date: "5/13/2022"
+output: html_document
+---
+
+An important step in becoming more eco-friendly is to consume local and seasonal produce.  
+This package is meant to make the process of eco-conscious consumption easier.  
+  
+The package will consist of different functions all designed to return information about seasonal food. The main function "whatsinseason()" will require 2 arguments: time and place, and it will return a list of produce that is in season for that time and place. 
+
+Here is a simple example: 
+```{r, include=FALSE}
+source("R/in_season_test.R")
+```
+
+```{r}
+whatsinseason("summer", "Netherlands")
+```
+
+### Ideas for further functions that can be built into this package:  
```

Comment:
I'm wondering if you could also expand the base function to allow the user to specify the place and season in different formats? Some countries might, for example, have variation in seasonal products due to area-wise variation in weather/soil conditions, so specifying the entire country might be misleading.